auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -568,6 +578,18 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     }
   }
 
+  showBusinessUnavailableModal (action: string) {
+    this.dialogTitle = 'Business unavailable'
+    this.dialogText = 'You are not authorized to access the business'
+    if (action === 'change name') {
+      this.dialogText += ' to change its name'
+    } else {
+      this.dialogText += ' you wish to ' + action
+    }
+    this.dialogText += '. Please add this business to your table to continue.'
+    this.$refs.businessUnavailableDialog.open()
+  }
```

Comment:
Please show screenshots of the 2 variations of this new dialog.

Reply:
#### The new dialog: 
![image](https://github.com/bcgov/sbc-auth/assets/60866283/f64d6ece-f2aa-4fd9-b0f9-3e3ac24fd13f)

The wording is slightly different according to different NR types